### ISSUE TITLE
Leapp-lock.json got corrupted, would not allow me to add a SAML 2.0 URL until Leapp-lock was moved

### ISSUE THREAD

**Issue by Gowiem** [Noovolari/leapp#302]
**Describe the bug**
I was continuing to get "Cannot read properties of null (reading 'id')" and "Cannot read properties of null (reading 'url')" errors when trying to add an SAML 2.0 URL through both possible flows. This continued until I moved my Leapp-lock.json and reopened the application. 

**Leapp Version**
0.13.1

**To Reproduce**
I don't know. It must have gotten corrupted somehow 😢 

**Expected behavior**
It would be great if the Application could deal with corrupted Leapp-lock.json files somehow. Or if we had a utility to decrypt and fix the JSON ourselves. 

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Mac
 - OS Version: 12.5
 - Leapp Version: 0.13.1

**Additional context**
N/A

**Comment by andreacavagna01** [comment]
We are sorry to hear that, but this is a great idea. 

[fig](https://fig.io) is doing the same with `fig doctor`.

Thanks for the contribution, soon we will look into it!

**Comment by Gowiem** [comment]
@andreacavagna01 Yeah, a `doctor` command would be great. It would help all these folks as well: https://github.com/Noovolari/leapp/issues/282
